src/AppBundle/Controller/AdminController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Admin;
+use AppBundle\Form\AdminType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+/**
+ * Class DefaultController
+ * @Route("/admin", name="admin")
+ * @package AppBundle\Controller
+ */
+class AdminController extends Controller
+{
+    /**
+     * @Route("/", name="admin_info")
+     * @Template("@App/Admin/index.html.twig")
+     *
+     * @param Request $request
+     * @return array
+     */
+    public function indexAction(Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $admin = $em->getRepository(Admin::class)->findOneBy(['id' => $this->getUser()->getId()]);
+        $editForm = $this->createForm(AdminType::class, $admin)
+            ->add('Save', SubmitType::class, array(
+                'attr' => array('class' => 'btn btn-primary')
+            ));
+        $editForm->handleRequest($request);
+        if ($editForm->isSubmitted()) {
```

Comment:
Call Form::isValid() with an unsubmitted form is deprecated since version 3.2 and will throw an exception in 4.0. Use Form::isSubmitted() before Form::isValid() instead.